### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by jmklix**
*Description of changes:*
standardized README format
moved examples to sample folder README

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 Amazon Web Services
```
Comment (justinboswell) [inline LICENSE:190]: Let's get this to match the copyright from all the other repos

File: README.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 # AWS IoT SDK for C++ v2
-
-Next generation AWS IoT Client SDK for C++.
+This document provides information about the AWS IoT device SDK for C++ V2.
 
 This project is in **GENERAL AVAILABILITY**. If you have any issues or feature
```
Comment (justinboswell) [inline README.md:4]: This first sentence is no longer necessary

File: README.md [hunk 3]
```diff
@@ -38,288 +44,59 @@ cd aws-iot-device-sdk-cpp-v2-build
...
-        "iot:Subscribe"
-    ],
-    "Resource": [
-        "arn:aws:iot:<your-region>:<your-id>:topicfilter/a/b"
-    ]
-}
-```
+*   Check api and developer guides.
+*   Check for similar issues already opened.
+*   Gitter or whatever chat app we implement.
```
Comment (justinboswell) [inline README.md:70]: can probably just leave this out until we have a link :)

File: LICENSE (hunk 1 above)
Comment (jmklix) [reply LICENSE:190]: Should I add both lines or just the copyright?

Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
SPDX-License-Identifier: Apache-2.0

File: samples/README.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+ 
+```
+./fleet-provisioning --endpoint <endpoint> --ca_file <path to root CA> 
+--cert <path to the certificate> --key <path to the private key> 
+--template_name <template name> --template_parameters <template parameters json>
+```
+
+Run the sample like this to provision using Csr:
+ 
+```
```
Comment (TingDaoK) [inline samples/README.md:85]: Fenced code blocks should have a language specified. So, should it be ...sh ?

File: README.md [hunk 5]
```diff
@@ -38,288 +43,58 @@ cd aws-iot-device-sdk-cpp-v2-build
...
-</details>
-
-## Greengrass Discovery
-This sample is intended for direct usage with the Greengrass tutorial found [here](https://docs.aws.amazon.com/greengrass/latest/developerguide/gg-gs.html).
+and [Jobs](https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html)
+is provided by code that been generated from a model of the service.
+
+
 
 # License
```
Comment (TingDaoK) [inline README.md:98]: Can you help to change the license to second-tier heading?

File: samples/README.md [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+      "Action": "iot:Connect",
+      "Resource": "arn:aws:iot:<b>region</b>:<b>account</b>:client/samples-client-id"
+    }
+  ]
+}
+</pre>
+</details>
+
+## Greengrass Discovery
+This sample is intended for direct usage with the Greengrass tutorial found [here](https://docs.aws.amazon.com/greengrass/latest/developerguide/gg-gs.html).
```
[the remark this answers is not in the document]
Comment (jmklix) [reply samples/README.md:288]: I have added it, but is there a purpose for a new line at then end of file? or is it just best practice?

[the remark this answers is not in the document]
Comment (TingDaoK) [reply samples/README.md]: Just for formatting...

**Pull request merged**